src/pages/gateway/command-reference.md

```diff
@@ -401,6 +401,70 @@ aio api-mesh:source:get -m -s "AEM Assets API" -s "Adobe Target API"
 ]
 ```
 
+## aio api-mesh:source:install
+
+Updates the currently selected workspace's mesh configuration with the selected configuration from the specified source.
+
+<InlineAlert variant="info" slots="text"/>
+
+Sources are prebuilt mesh configuration files that are formatted for a specific combination of handlers. Each source contains a mesh configuration file designed for a specific first or third-party source. Third-parties can submit their sources as a pull request to the [api-mesh-sources](https://github.com/adobe/api-mesh-sources) repo. See [Create a mesh from a source](./create-mesh.md#create-a-mesh-from-a-source) for more information.
+
+### Usage
+
+```bash
+aio api-mesh:source:install "SOURCE_NAME"
```

Comment:
Change "SOURCE_NAME" to `<source_name>` or `<Source_Name>`

Edit: The usage statement for `aio api-mesh:source:get` ends with %SOURCE_NAME%, so that's a different format. They should probably both use the less/greater than symbols